Makefile

```
@@ -3,6 +3,17 @@ BASE_BRANCH ?= devel
 PROTOC_VERSION=3.17.3
 export BASE_BRANCH
 
+# Define LOCAL_BUILD to build directly on the host and not inside a Dapper container
+ifdef LOCAL_BUILD
+DAPPER_HOST_ARCH ?= $(shell go env GOHOSTARCH | cut -d= -f2 | tr -d '"')
```

```suggestion
DAPPER_HOST_ARCH ?= $(shell go env GOHOSTARCH)
```

`go env GOHOSTARCH` only outputs that specific value, not `GOHOSTARCH="foo"` (as with `go env`).

---

thanks for the catch on `go env` - amended
will submit the print-% target on shipyard